src/shared/amplify.config.ts

```diff
@@ -0,0 +1,37 @@
+import { AmplifyConfig } from "shared/interfaces/amplify.interface";
+
+export const amplifyConfig: AmplifyConfig = {
+  Auth: {
+    identityPoolId: process.env.REACT_APP_IDENTITY_POOL_ID,
+    region: process.env.REACT_APP_REGION,
+    userPoolId: process.env.REACT_APP_USER_POOL_ID,
+    userPoolWebClientId: process.env.REACT_APP_USER_POOL_WEBCLIENT_ID,
+  },
+  language: 'us'
+};
```

Comment:
The AmplifyConfig object is based on the amplify documentation for manuel setup: https://aws-amplify.github.io/docs/js/authentication#manual-setup